CONTRIBUTING.md

```diff
@@ -96,6 +96,10 @@ Sorry, no pseudonyms or anonymous contributions are allowed.
 We also require each commit be individually signed-off by their author,
 even when part of a larger set. You may find `git commit -s` useful.
 
+## Code of Conduct
+
+When contributing, you must adhere to the Code of Conduct, which is available at: https://github.com/lxc/lxd/blob/master/CODE_OF_CONDUCT.md
```

Comment:
Can we make this a relative link?

Maybe something like:
```
When contributing, you must adhere to the [Code of Conduct](CODE_OF_CONDUCT.md).
```

Reply:
As mentioned in the commit, I added it as absolute link so that we can reuse the text and the link will work both directly from the file and from the docs. 
Of course it's not a lot of text, so we can also duplicate it, but I figured that we probably always want contributors to adhere to the latest Code of Conduct anyway?

Reply:
Ah, didn't see the commit message on that one, yeah, that's probably fine.